peachjam/js/components/LegislationTable/index.vue

```diff
@@ -0,0 +1,319 @@
+<template>
+  <div class="row">
+    <div
+      v-if="showSideFilters"
+      class="col col-lg-3"
+    >
+      <SecondaryFacets v-model="facets" />
+    </div>
+    <div
+      class="col"
+    >
+      <div class="card legislation-table">
+        <div class="card-header">
+          <div class="input-group">
+            <button
+              class="btn btn-outline-secondary dropdown-toggle"
+              type="button"
+              data-bs-toggle="dropdown"
+              aria-expanded="false"
+            >
+              Search by {{ filters[selectedFilterKey].label }}
```

Comment:
Two thoughts:

1. Avoid the word "search", which can imply lots of smarts; rather use "filter", which is clearer that it's going to look for exactly what you type without any smarts
2. can we reduce this to just one input box, and the same filter is applied to both title fields, and if any match then the row is retained. The benefit of this is the user doesn't have to think: typing "Children" and "5 of 2020" both do what the user expects.